### PR TITLE
#2061 get all design reviews endpoint

### LINKED ISSUE #2061: Design Review Calendar - Get All Design Reviews Endpoint
### Description

Blocked by: #1846

We need a way to access all of the design reviews in our database on Finishline. This ticket is to create a backend endpoint that retrieves all design reviews.

### Acceptance Criteria

- [ ] Create an endpoint to access all design reviews
- [ ] It should be on the route '/design-reviews/'
- [ ] Create unit tests
- [ ] Manually test using Postman and include screenshots of the query result

### Proposed Solution

Take a look at some other "get all" endpoints

### Mocks

_No response_

### PR DESCRIPTION
**Pull request by jessicayzhao**
## Changes

Get all design reviews endpoint

## Test Cases

Get All Design Reviews works when there are change requests

## Screenshots

_If you did any manual testing (e.g., with Postman), put screenshots of the http request and before and after of the db_

Screenshot 2024-02-28 at 3 41 10 AM

Screenshot 2024-02-29 at 11 58 45 PM

## To Do

_Any remaining things that need to get done_

- [x] backend tests
- [x] postman tests

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes #2061

### CODE REVIEW

**Review by walker-sean — CHANGES_REQUESTED** [review]
Looking good! 

Can you also add some design reviews locally (you can use yarn prisma:studio) to show that the endpoint works with a populated database for manual testing?

File: src/backend/src/controllers/design-review.controllers.ts [hunk 1]
```diff
@@ -4,6 +4,15 @@ import { getCurrentUser } from '../utils/auth.utils';
 import DesignReviewService from '../services/design-review.services';
 
 export default class DesignReviewController {
+  static async getAllChangeRequests(_req: Request, res: Response, next: NextFunction) {
```
Comment (Aaryan1203) [inline src/backend/src/controllers/design-review.controllers.ts:7]: I think this should be getAllDesginReviews

File: src/backend/src/routes/design-review.routes.ts [hunk 2]
```diff
@@ -3,6 +3,8 @@ import DesignReviewController from '../controllers/design-review.controllers';
 
 const designReviewRouter = express.Router();
 
+designReviewRouter.get('/', DesignReviewController.getAllChangeRequests);
```
Comment (Aaryan1203) [inline src/backend/src/routes/design-review.routes.ts:6]: Even over here

**Review by Aaryan1203 — CHANGES_REQUESTED** [review]
I was just looking through this PR and left a couple comments!

File: src/backend/src/controllers/design-review.controllers.ts (hunk 1 above)
Comment (jessicayzhao) [reply src/backend/src/controllers/design-review.controllers.ts:7]: haha oops

File: src/backend/src/controllers/design-review.controllers.ts [hunk 3]
```diff
@@ -4,6 +4,15 @@ import { getCurrentUser } from '../utils/auth.utils';
 import DesignReviewService from '../services/design-review.services';
 
 export default class DesignReviewController {
+  static async getAllDesignReviews(_req: Request, res: Response, next: NextFunction) {
+    try {
+      const changeRequests = await DesignReviewService.getAllDesignReviews();
```
Comment (walker-sean) [inline src/backend/src/controllers/design-review.controllers.ts:9]: This should say design reviews

File: src/backend/src/services/design-review.services.ts [hunk 4]
```diff
@@ -6,6 +6,18 @@ import designReviewQueryArgs from '../prisma-query-args/design-review.query-args
 import { designReviewTransformer } from '../transformers/design-review.transformer';
 
 export default class DesignReviewService {
+  /**
+   * Gets all design reviews in the database
+   * @returns All of the design reviews
+   */
+  static async getAllDesignReviews(): Promise<DesignReview[]> {
+    const designReview = await prisma.design_Review.findMany({
```
Comment (walker-sean) [inline src/backend/src/services/design-review.services.ts:14]: make this plural please

**Review by walker-sean — CHANGES_REQUESTED** [review]
Nice! I think you'll be good to go after the extra files are reverted

**Review by walker-sean — APPROVED** [review]
Nice!

**Pull request merged**